release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -536,6 +536,13 @@ For more information, see xref:../networking/dns-operator.adoc#nw-controlling-dn
 ==== Provider networks support for clusters that run on {rh-openstack}
 {product-title} clusters on {rh-openstack-first} now support provider networks for all deployment types.
 
+[id="ocp-4-8-configurable-headerBufferMaxRewriteByte-headerBufferBytes-parameters"]
+==== Configurable tune.maxrewrite and tune.bufsize for HAProxy
+
+Cluster Administrators can now set `headerBufferMaxRewriteByte` and `headerBufferBytes` Ingress Controller tuning parameters to configure `tune.maxrewrite `and `tune.bufsize` HAProxy memory options per-Ingress Controller.
```

Comment:
might just be me, but I was thrown a little by the ID= value *not* matching the heading.

Also, maybe use two sentences?  "...Controller tuning parameters.  These parameters correspond to the...`

That sentence was a lot for this newb to take in at one time.